source/gui/settingsDialogs.py

```diff
@@ -1239,6 +1247,7 @@ def onSave(self):
 		config.conf["speech"][synth.name]["capPitchChange"]=self.capPitchChangeEdit.Value
 		config.conf["speech"][synth.name]["sayCapForCapitals"]=self.sayCapForCapsCheckBox.IsChecked()
 		config.conf["speech"][synth.name]["beepForCapitals"]=self.beepForCapsCheckBox.IsChecked()
+		config.conf["speech"][synth.name]["readNumbersAs"]=self.digitsCombo.GetSelection()

```

Comment:
There is a try/except block around the call of SetSelection, since the configuration could theoretically have a value that is out of range for the combo box. When saving the value, the combobox will always have a value that is in range for the config. I don't think another try/except is necessary here.